docker/test-gpu-local.sh

```diff
@@ -45,5 +45,4 @@ docker run \
     graphistry/test-gpu:${TEST_CPU_VERSION} \
         --maxfail=1 \
         --ignore=graphistry/tests/test_feature_utils.py \
-        --ignore=graphistry/tests/test_umap_utils.py \
         $@
```

Comment:
shouldn't we be testing umap, as that's the big point of the release?

Reply:
It's ignoring feature_utils only